rts/at/integration/src/main/java/org/jboss/narayana/rest/integration/RecoveryManager.java

```diff
@@ -232,14 +235,14 @@ private Participant recreateParticipant(final InputObjectState inputObjectState,
 
     private boolean synchronizeParticipantUrlWithCoordinator(final ParticipantInformation participantInformation) {
         final String participantUrl = getParticipantUrl(participantInformation.getId());
-        final Link participantLink = new Link(TxLinkNames.PARTICIPANT_RESOURCE, TxLinkNames.PARTICIPANT_RESOURCE,
-                participantUrl, null, null);
-        final Link terminatorLink = new Link(TxLinkNames.PARTICIPANT_TERMINATOR, TxLinkNames.PARTICIPANT_TERMINATOR,
-                participantUrl, null, null);
+
+        Client client = ClientBuilder.newClient();
```

Comment:
This could just be created when it's needed. This variable doesn't seem to be reused.